File: ws/CEDENOR/U2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Fish.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
} * @author Nahomi Cedeño, codecreators, DCCO-ESPE
 */
public abstract class Fish extends Animal{
    
    public Fish(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }
    
}

File: hw/travez/u2/HW18ZooInPython/ec_edu_espe_zoo_model/Fish.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from ec_edu_espe_zoo_model.Animal import Animal

# Ismael Travez, The Java Bandits, DCCO-ESPE


class Fish(Animal):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)
